app/src/main/java/com/keylesspalace/tusky/entity/Instance.kt

```diff
@@ -19,19 +19,20 @@ import com.google.gson.annotations.SerializedName
 
 data class Instance(
     val uri: String,
-    val title: String,
-    val description: String,
-    val email: String,
+    // val title: String,
+    // val description: String,
+    // val email: String,
```

Comment:
What's up with this? Just cleaning up fields we don't use?

Reply:
Removing unused fields speeds up json deserializing and uses a bit less memory